src/model/status.nit

```diff
@@ -119,6 +124,21 @@ class MissionStatus
 	var is_success: Bool is lazy do return status == "success"
 end
 
+# The link between a Player and a Star
+class StarStatus
+	super Entity
+	serialize
+
+	# The associated star
+	var star: MissionStar
+
+	# Is the star granted?
+	var is_unlocked = false is writable
+
+	# The current best score (for ScoreStar)
+	var best_score: nullable Int = null is writable
```

Comment:
maybe latter. I do not want to add date/time-format issues and discussion in the PR.
We can also keep the specific submission that get the best score (for the user pleasure).
StarStatus will be improved by other PR I suppose